src/frameworks/vue/index.js

```diff
@@ -1,3 +1,35 @@
+const inquirer = require('inquirer');
+const path = require('path');
+
+const dependencies = {
+  devDependencies: {
+    'sass-loader': '^10.0.3',
+    'style-resources-loader': '^1.3.3',
+  },
+};
+
 module.exports = async function vue() {
-  this.log('todo');
+  const { projectName } = await inquirer.prompt({
+    type: 'input',
+    name: 'projectName',
+    message: 'Enter the folder name for your project',
+  });
+
+  await this.asyncCommand('vue', ['create', projectName]);
+  process.chdir(projectName); // change directory to new folder
+
+  this.log('Copying template files');
+  await this.copy(
+    path.join(__dirname, './template'),
+    this.destinationPath(),
+  );
+
+  this.log('Copying common files');
```

Comment:
If we copy common files first, it would allow template files to overwrite them. That might come in handy later.